logs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
A .gitignore was included inside the **logs** folder in order to include it in git, and [exclude all its contents at the same time](http://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository). The logger will fail if it doesn't find this folder.